collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -274,20 +273,11 @@ private boolean shouldRowBeInserted(HashMap<String, String> answers) {
         return false;
     }
 
-    private String uploadMediaFile(File instanceFile, String fileName)
-            throws UploadException {
+    private String uploadMediaFile(File instanceFile, String fileName) throws UploadException {
         String filePath = instanceFile.getParentFile() + "/" + fileName;
         File toUpload = new File(filePath);
 
-        // first check the local content provider
-        // to see if this photo still exists at the location or not
-        String selection = MediaStore.Images.Media.DATA + "=?";
-        String[] selectionArgs = {filePath};
-        Cursor cursor = Collect.getInstance().getContentResolver()
-                .query(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, null, selection,
-                        selectionArgs, null);
-        if (cursor != null && cursor.getCount() != 1) {
-            cursor.close();
+        if (!new File(filePath).exists()) {
```

Comment:
To help reviewers, would you please explain this change and how it fixes the problem? Thanks.